static/js/main.js

```diff
@@ -1,33 +1,62 @@
 "use strict";
 
-(function()
-{
-  function initLanguageSelection()
-  {
-    var locale = document.getElementById("navbar-locale-selected");
-
-    // skip if page does not have language selection (EG: blog)
-    if (!locale) return;
-
-    locale.addEventListener("click", function()
-    {
-      document.getElementById("navbar-locale-menu")
-        .classList.toggle("visible");
-    }, false);
-  }
-
-  function navigationClick(event)
-  {
-    document.getElementById("navbar-menu")
-      .classList.toggle("visible");
-  }
-
-  function initMenu()
-  {
-    document.getElementById("navbar-menu-toggle")
-      .addEventListener("click", navigationClick, false);
-  }
-
-  initLanguageSelection();
-  initMenu();
+(function() {
```

Comment:
About your first comment - no problem, I actually really appreciate that. 
And about this one - how do I actually use the eslint and stylelint in this project? I followed the instructions, but I don't see that it works (installed it globally, trying to use it on a specific file etc)

Reply:
Good question. Will you take this one @ireade? 

Anyway, you may ignore this issue for now. On second thought, I'm not sure that it makes sense to nit-pick contributors about coding style issues. We can probably prettify your code to match our coding style when we merge it.

Reply:
We are currently working on getting our `stylelint-config-eyeo` on npm, so unfortunately right now it can't be installed globally. 

However, you can install/configure it by manually downloading the repo and referencing it in your `stylelintrc.json` file: 

```json
{
  "extends": "path/to/codingtools/stylelint-config-eyeo"
}
```

But as Julian said, you can probably ignore the styling issues for now / let us handle it.